setup.cfg

```diff
@@ -1,24 +1,58 @@
 [metadata]
-package_name = ci_watson
+name = ci_watson
 description = CI helper for STScI Jenkins
 author = STScI
 author_email = help@stsci.edu
 license = BSD
+license_file = LICENSE.md
 url = https://github.com/spacetelescope/ci_watson
+long_description = file: README.md
+long_description_content_type = text/markdown
 classifier =
     Framework :: Pytest
     Intended Audience :: Science/Research
     License :: OSI Approved :: BSD License
     Operating System :: OS Independent
     Programming Language :: Python :: 3
+    Programming Language :: Python :: Implementation :: CPython
     Topic :: Software Development :: Testing
     Topic :: Software Development :: Libraries :: Python Modules
 
+[options]
+# We set packages to find: to automatically find all sub-packages
+packages = find:
+include_package_data = True
+zip_safe = False
+setup_requires = setuptools_scm
+install_requires =
+    pytest>=3
+    requests
+python_requires = >=3.5
+
+[options.packages.find]
+exclude =
+    tests
+
+[options.extras_require]
+all =
+    astropy
```

Comment:
`astropy` is also a test dependency and should be included there.

Reply:
Which might negate the need for `all` alltogether.

Reply:
I moved it to `all` because there is usage of `astropy` in the helpers code. `ci-watson` basically won't be useful without `astropy`, not just for testing. Does this make sense or did I misunderstand `extras_require`?

Reply:
If I want to run the package tests, `astropy` is required.  Thus is should be in `[tests]`.  If you also want to include it in `all`, that's fine too I think.  👍

Reply:
Okay, added it back to `test`. Thanks for the review!